pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -261,6 +262,22 @@ func (r *ReconcilePerconaServerMongoDB) updateConfigMembers(ctx context.Context,
 			return 0, fmt.Errorf("get host for pod %s: %v", pod.Name, err)
 		}
 
+		nodeLabels := mongo.ReplsetTags{
+			"nodeName":    pod.Spec.NodeName,
+			"podName":     pod.Name,
+			"serviceName": cr.Name,
+		}
+
+		labels, err := psmdb.GetNodeLabels(ctx, r.client, cr, pod)
+		if err != nil {
+			log.Info("failed to get node labels, continue without node labels", "pod", pod.Name, "error", err.Error())
+		} else {
+			nodeLabels = util.MapMerge(nodeLabels, mongo.ReplsetTags{
+				"region": labels[corev1.LabelTopologyRegion],
+				"zone":   labels[corev1.LabelTopologyZone],
+			})
+		}
```

Comment:
let's add labels only if `err == nil`. otherwise this will pollute the logs constantly

Reply:
done, thanks.